src/mutation.cpp

```diff
@@ -556,6 +556,10 @@ trait_id Character::trait_by_invlet( const long ch ) const
 
 bool player::mutation_ok( const trait_id &mutation, bool force_good, bool force_bad ) const
 {
+    if (mutation_branch::trait_is_blacklisted(mutation)) {
+        // This mutation has been blacklisted.
```

Comment:
Well, isn't that what the condition above already says? What else should the return value `true` of `mutation_branch::trait_is_blacklisted` mean? Why add a comment for this obvious behaviour?